pages/1.11/release-notes/1.11.0/index.md

```diff
@@ -0,0 +1,138 @@
+- Fault domain awareness for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic and DC/OS HDFS. Use fault domain awareness to make your services highly available and to allow for increased capacity when needed.
+- New API endpoint to pause a node for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS. Use this endpoint to relaunch a node in an idle command state for debugging purposes.
+- New beta DC/OS Kafka ZooKeeper service. [View the documentation](/services/beta-kafka-zookeeper).
+- You can now select a DC/OS data service version from a dropdown menu in the DC/OS UI.
+- Improved scalability for all DC/OS data services.
+- Non-essential tasks now supported for DC/OS Kafka, DC/OS Cassandra, DC/OS Elastic, and DC/OS HDFS. If non-essential tasks fail, the entire node will not fail.
+
+## <a name="known-issues"></a>Known Issues and Limitations
+- DCOS-9751	- Marathon fails to authenticate with Mesos master during disabled -> permissive upgrade.
+- DCOS-18368 - The GUI installer has been retired in 1.11 and will no longer continue to function. It will be decommissioned in 1.12. For details of alternative installation methods, [view the documentation](https://docs.mesosphere.com/1.11/installing).
+- DCOS-19047 - `dcos-secrets` service is unavailable during upgrade from 1.10.x to 1.11. 
+- DCOS_OSS-2132	- `dcos-log` does not handle the journald files rotation properly.
+- INFINITY-3116	- Deleting failed mnist Tensorflow package never completes.
+
+## <a name="fixed-issues"></a>Improvements and Major Issues Fixed in 1.11.0
+- COPS-2201 - `dcos-diagnostics` no longer fails after running gen_resolvconf.py.
+- DCOS-13066 - Rename the 3DT service to DC/OS Diagnostics.
+- DCOS-13325 - Added domain awareness to DC/OS installers.
+- DCOS_18165 - Support DC/OS on non-root LVM volumes.
+- DCOS-18981 - Add a CLI command to mark an agent as gone.
+- DCOS-19201 - Add binary data support to dcos security CLI.
+- DCOS-19837 - Consolidate fault-domain scripts for all cloud providers into one script.
+- DCOS-19896 - Add `--linked` flag to `dcos cluster list`.
+- DCOS-20351 - Add the `dcos-license` subcommand to `dcos-enterprise-cli`. [enterprise type="inline" size="small" /]
+- DCOS-21130 - Make list of services visible in grid view of nodes. 
+- DCOS_OSS-671 - Improve filenames in the diagnostics bundle.
+- DCOS_OSS-1275 - Add support for custom check executables during on-premises installation.
+- DCOS_OSS-1321 - Search paths are configurable when executing check commands using `dcos-diagnostics check`.
+- DCOS_OSS-1340 - Spartan "autoip" DNS resolves to host IP for UCR in bridge network.
+- DCOS_OSS-1449 - Remove support for `--appId` from CLI.
+- DCOS_OSS-1489 - Add support for cgroup blkio statistics in `dcos-metrics`.
+- DCOS_OSS-2003 - Modify DC/OS overlay networking to work with systemd networkd.
+
+## <a name="fixed-issues"></a>Improvements and Major Issues Fixed in 1.11.0 Release Candidate 4
```

Comment:
Is that supposed to be the "diff" between RC4 and RC1?

Where do we want to show the changes that happened between RC4 and RC6?